code/modules/organs/organ_internal.dm

```diff
@@ -204,20 +224,16 @@
 
 	if(owner.reagents.get_reagent_amount(/datum/reagent/medicine/peridaxon) >= 0.05)
 		return
-	if(is_bruised())
-		if(prob(5))
-			owner.emote("me", 1, "coughs up blood!")
-			owner.drip(10)
-		if(prob(15))
-			owner.emote("me", 1, "gasps for air!")
-			owner.Losebreath(4)
-	else if(is_broken())
-		if(prob(30))
-			owner.emote("me", 1, "coughs up blood!")
-			owner.drip(10)
-		if(prob(50))
-			owner.emote("me", 1, "gasps for air!")
-			owner.Losebreath(10)
+	if((organ_status == ORGAN_BRUISED && prob(15)) || (organ_status == ORGAN_BROKEN && prob(50)))
```

Comment:
prob(50) on life is gonna be real spammy when broken lungs don't actually kill you, maybe 5 and 20?